lib/xclarity_client/services/xclarity_service.rb

```diff
@@ -34,14 +34,15 @@ def get_object(uuids, includeAttributes, excludeAttributes)
         $lxca_log.info "XclarityClient::Endpoints::XClarityService get_object",
                        "Sending request to #{managed_resource} resource"
 
-        uuids.reject! { |uuid| UUID.validate(uuid).nil? } unless uuids.nil?
+        uuid_array = uuids.reject { |uuid| uuid.nil? || uuid.empty? } unless uuids.nil?
```

Comment:
Here you can just do `uuids&.reject` and remove the `unless uuids.nil?` and you will get the same effect

Reply:
You're right, fixing this now.